spec/Procedures.tex

```diff
@@ -1295,38 +1295,46 @@ \subsection{Determining More Specific Functions}
  Otherwise neither mapping is more specific.
 \end{itemize}
 
+\subsection{Determining Best Function}
+\label{Determining_Best_Function}
+
+Given two functions $F_1$ and $F_2$, the better function is
+determined by the following steps:
+
+\begin{itemize}
+\item If $F_1$ is more specific than $F_2$, then $F_1$ is a better match.
+\item If $F_2$ is more specific than $F_1$, then $F_2$ is a better match.
+\item If $F_1$ has a where clause and $F_2$ does not have a where clause,
+then $F_1$ is a better match.
+\item If $F_2$ has a where clause and $F_1$ does not have a where clause,
+then $F_2$ is a better match.
+\end{itemize}
```

Comment:
Let's use "the better function" instead of "a better match" in the four cases above.